src/partition/ReceivedPartition.cpp

```diff
@@ -482,6 +476,11 @@ void ReceivedPartition::createOwnerInformation()
   }
 }
 
+bool ReceivedPartition::hasVerticesAtCouplingInterface() const {
+    return (_fromMapping && not _fromMapping->getOutputMesh()->vertices().empty()) ||
+           (_toMapping   && not _toMapping->getInputMesh()->vertices().empty());
+}
+
```

Comment:
Good effort, but the name might be misleading (so was probably the comment before). It's not **this** mesh (the mesh associated to this partition), but the other meshes (connected via mappings) that have vertices at the coupling interface. 
Maybe `areConnectedProvidedMeshesAtCouplingInterface` ?

Reply:
Misleading comment indeed.

This name looks a bit brutal. What about `isRemoteMeshEmpty`?

Reply:
* `remote`, we never use for meshes, and would again be misleading, because one could argue that the received mesh is the remote one. The right expression here would be `provided`. 
* It can be one or two meshes. 

`areProvidedMeshesEmpty` ?

Reply:
Nice! Short and expressive.